idaes/core/base/flowsheet_model.py

```diff
@@ -275,9 +275,10 @@ def visualize(self, model_name, **kwargs):
             **kwargs: Additional keywords for :func:`idaes.core.ui.fv.visualize()`
 
         Returns:
-            None
+            VisualizeResult : The return from UI, namedtuple("VisualizeResult", ["store", "port", "server", "save_diagram"]) etc...
         """
-        UI().visualize(self, model_name, **kwargs)
+        VisualizeResult = UI().visualize(self, model_name, **kwargs)
+        return VisualizeResult
```

Comment:
The convention in Python is to use `snake_case` for variables/instances, and `PascalCase` for classes. Since the value returned here is an instance, I think `snake_case` is the appropriate one:

```suggestion
        visualize_result = UI().visualize(self, model_name, **kwargs)
        return visualize_result
```